src_app/test_scenes/index.js

```diff
@@ -8,16 +8,20 @@ export * from './sceneColorPicker';
 export * from './sceneDatePicker';
 export * from './sceneDialog';
 export * from './sceneDropdownList';
+export * from './sceneFilePicker';
 export * from './sceneGridLayout';
 export * from './sceneImage';
-export * from './sceneImageSlider/scene';
 export * from './sceneImageRemote';
+export * from './sceneImageSlider/scene';
```

Comment:
We don't need to provide `/scene`:

```export * from './sceneImageSlider';```

Reply:
Looks like post rebase leftover - cleaned-up.